docs/_coverpage.md

```diff
@@ -7,6 +7,6 @@
 - <span class='cover-icon' style="color: #002ead;"><i class="fad fa-question-circle"></i></span> [Help by answering coding questions](https://www.freecodecamp.org/forum/c/help?max_posts=1) on our community forum.
 - <span class='cover-icon' style="color: #00471b;"><i class="fad fa-comments-alt"></i></span> [Give feedback on coding projects](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1) built by campers.
 - <span class='cover-icon' style="color: #c4302b;"><i class="fab fa-youtube"></i></span> [Help us add subtitles (closed captions)](https://www.youtube.com/timedtext_cs_panel?c=UC8butISFwT-Wl7EV0hUK0BQ&tab=2) to our YouTube channel videos.
-- <span class='cover-icon' style="color: #000000;"><i class="fab fa-github"></i></span> [Contribute to our open source codebase](/index?id=our-open-source-codebase) on GitHub.
+- <span class='cover-icon git'><i class="fab fa-github"></i></span> [Contribute to our open source codebase](/index?id=our-open-source-codebase) on GitHub.
```

Comment:
What is this change? Is it intentional?

Reply:
Yes, it was. When switch to dark mode, git icon become invisible. So to change the color of git icon.

Reply:
We probably should take care to match GitHub's colours.  From using a colour picker on GitHub itself, it looks like they use white and transparent for the light icon and #171515 and transparent for dark.  Not black, but it is close.